test/unit/wdpa/pame_importer_test.rb

```diff
@@ -16,9 +16,9 @@ class TestPameImporter < ActiveSupport::TestCase
     assert_equal 9, pame_evaluations.count
   end
 
-  test "#import pame evaluations with hidden evaluation" do
-
-    PAME_EVALUATIONS = "#{Rails.root}/lib/data/seeds/test_pame_data_hidden.csv".freeze
+#  test "#import pame evaluations with hidden evaluation" do
+  def badger
```

Comment:
What is this about exactly?

Reply:
badly skipped test - I have fixed test and extended the expectations to check the three restricted cases here